README.md

```diff
@@ -110,3 +110,14 @@ go to [this folder](man)
 - FAU grey: R140, G159, B177
 - green: R0, G255, B0
 - blue: R0, G0, B127
+
+### VTK export
+
+The export function of the subsystem returns a tuple containing `(points, cells, point_data, cell_data)`:
+
+- **points**: A list of 3-dimensional NumPy arrays, each representing the coordinates of a point in space.
+- **cells**: A list of tuples, where each tuple consists of a VTK cell type (`VTK_CELL_TYPE`) and the associated connectivity information. The `VTK_CELL_TYPE` defines the type of geometric cell, and connectivities define how points are connected within that cell. For reference, some useful `VTK_CELL_TYPE` examples can be found at:
+  - [Linear Cells](https://examples.vtk.org/site/Cxx/GeometricObjects/LinearCellsDemo/)
+  - [Isoparametric Cells](https://examples.vtk.org/site/Cxx/GeometricObjects/IsoparametricCellsDemo/)
```

Comment:
I'd link the Python examples

https://examples.vtk.org/site/Python/GeometricObjects/LinearCellsDemo/

https://examples.vtk.org/site/Python/GeometricObjects/IsoparametricCellsDemo/

A list of all VTK constants can be found here: https://gitlab.kitware.com/vtk/vtk/-/blob/ce05c6993d68cc9c444a70b44615e771738fbefb/Wrapping/Python/vtkmodules/util/vtkConstants.py